SpaceDome/src/game.cpp

```diff
@@ -387,6 +390,10 @@ void Game::update() {
         it = mBullets.erase(it);
     }
 
+    for(auto it = boosters.begin(); it != boosters.end();) {
```

Comment:
Mer läsbart och enklare:
```suggestion
    boosters.clear();
```